dotcom-rendering/src/web/components/StickyBottomBanner.importable.tsx

```diff
@@ -226,7 +226,7 @@ export const StickyBottomBanner = ({
 
 	const asyncCountryCode = getLocaleCode();
 	const isSignedIn = !!getCookie({ name: 'GU_U', shouldMemoize: true });
-	const [SelectedBanner, setSelectedBanner] = useState<React.FC | null>(null);
+	const [SelectedBanner, setSelectedBanner] = useState<MaybeFC | null>(null);
```

Comment:
Where’s `MaybeFC` coming from?

Reply:
[It comes from `messagePicker`](https://github.com/guardian/dotcom-rendering/blob/6e87f8a27ecbbcd2acf15f94e7e95448ce19da9d/dotcom-rendering/src/web/lib/messagePicker.ts#L4).

I've used this as it is as tightly coupled to the upstream type as possible, but we might want to look at that code as a secondary PR as we can probably be more explicitly typed.

Reply:
I'm also not 100% sure why that code is passing, but it is.